pkg/types/ccipocr3/interfaces.go

```diff
@@ -21,7 +21,8 @@ type MessageHasher interface {
 }
 
 type ExtraDataCodec interface {
-	DecodeExtraData(ExtraArgs Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
+	DecodeExtraArgs(extraArgs Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
```

Comment:
nit: add comment on which exact field on the Message struct does this work on.

Reply:
We need comments on this entire struct, please add doc comments so this is understandable to people who maintain this codebase